vscode-wpilib/src/webviews/vscodestyling.ts

```diff
@@ -0,0 +1,55 @@
+const applyVsCodeStyling = () => {
```

Comment:
I think this would be cleaner/simpler if you had static .css files that you included in the webviews like in this example:

https://github.com/microsoft/vscode-extension-samples/blob/main/webview-sample/media/reset.css
https://github.com/microsoft/vscode-extension-samples/blob/main/webview-sample/media/vscode.css

And include them in the webviews like this: https://github.com/microsoft/vscode-extension-samples/blob/main/webview-sample/src/extension.ts#L201

Or maybe you can put those files in a `vscode-wpilib/resources/css` folder and include them input webviews like `projectcreator.html` by adding this to the header:

```
<link href="../css/reset.css" rel="stylesheet">
<link href="../css/vscode.css" rel="stylesheet">
```

Reply:
I tried going this way but using the relative path doesn't seem to work for me.
I think maybe we should use the styling from these files, but keep the way it is currently set up.
What do you think?

Reply:
Did you get some sort of console error? I'm not too familiar with this extension so I'd have to run it myself to see what the issue might be.